src/mongo/db/pipeline/accumulator.h

```diff
@@ -297,13 +331,21 @@ class AccumulatorAvg final : public Accumulator {
      */
     Decimal128 _getDecimalTotal() const;
 
+    // to be digested by TDigest algorithm
+    std::vector<double> values;
+
+    // process inputs will be sorted and added to this object
+    mongo::TDigest digest;
+
```

Comment:
The following members don't make much sense to me. Why do we keep them ?
* percentile
* _isDecimal
* _nonDecimalTotal
* _decimal_Total
* _count ( why do we keep track of the count of documents ?)

couldn't we remove them from this class ?

Reply:
Most of them removed. 
"_count" kept temporarily, we will talk about it.